src/index.html

```diff
@@ -7,6 +7,355 @@
     <link rel="stylesheet" href="./styles/main.scss">
```

Comment:
let's link the roboto font to follow the mockup

Reply:
+ fixed it